website/docs/02-About/3-branding.mdx

```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
 
-- **Main Color**: `#699b98`
-- **Accent Color 1**: `#808080`
+### Secondary Logomark
 
-The Accent Color is to be rarely used. The Main Color should be used across Font, Banners, and Menus.
+Shown here is the secondary Moja Global logomark.
 
-### Imagery
+The secondary logomark should be used carefully and in cases where the brand graphics are already present on the page.
 
-The images should be high-quality, engaging photography that portrays images of landscapes, forests, plants, and communities, and individuals interacting with the landscape.
+| Description  | Image |
+| ------------- |:-------------:|
+| Secondary Colorway 2A - is to be used in instances where a white/light colored background and/or a light colored image is present.      | <img src="/img/brand/secondary-logomark-2a.png" width="150"/> |
+| Secondary Colorway 2B is to be used in instances where a black/dark colored background and/or a light colored image is present.      | <img src="/img/brand/secondary-logomark-2b.png" width="150"/> |
 
-An example shown below is included in the [images](https://github.com/moja-global/About_moja_global/blob/master/images) directory. Please ensure that proprietary photographs are credited if the source is known.
+#### Monochromatic Usage
 
-<img src="/img/moja-global-banner.png" width="700" />
+Shown here are the monochromatic versions of the primary logomark. They are to be only used in cases where color print is restricted.
+
+| Description  | Image |
+| ------------- |:-------------:|
+| Secondary Colorway 2A - is to be used in instances where a white/light background is present.      | <img src="/img/brand/secondary-logomark-mono-2a.png" width="150"/> |
+| Secondary Colorway 2B - is to be used in instances where a black/dark background is present.      | <img src="/img/brand/secondary-logomark-mono-2b.png" width="150"/> |
+
+## Logomark Usage
+### Clear Space and Sizing
+Both the primary and secondary logos should always appear in their entirety, surrounded by a minimum area of clear space, so that headlines, text or other visual elements do not encroach on the logo. The minimum clear space is shown using the height of the brand icon. Logos should be used at a minimum size of 60px or 30mm (w).
+
+<img src="/img/brand/clear-space-and-sizing.png"/>
+
+### Logomark Misuse
+
+Do's and Don'ts
+
+Take care to maintain the integrity of the logomark. The rules demonstrated here apply to all versions of logos.
+
+Do not:
+
+1. Do not remove any asset of the logo
+2. Do not distort or skew the logo
+3. Do not modify the composition or its proportions
+4. Do not outline the typography or icon
+5. Do not add drop shadow/filters
+6. Do not frame the logo
+7. Do not use the logo on a background colour that renders any part of logo illegible.
+8. Do not angle the logo 09. Do not use the logo in an unspecified colour
+
+<img src="/img/brand/logomark-misuse.png"/>
```

Comment:
Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.